wrappers.py

```diff
@@ -46,7 +47,7 @@ def proxy_reward(self, reward, obs, done):
             return obs[0]
         elif self.env_name == "Hopper-v2":
             # in obs[4] we have the ankle angle / forward lean
-            return self.running_mean(obs[4]).mean if done else 0
+            return self.running_mean(obs[4]).mean() if done else 0
```

Comment:
no-wonder the proxy reward was zero.

Reply:
the proxy_reward was (before this modification) a numpy_object_something, and is now a float. It didn't actually run before this modification, so this was not causing anything to be zero.